### ISSUE TITLE
Peer review by Info crew

### ISSUE THREAD

**Issue by mausam20** [INFO-526-FA24/project-2-stat-squad#1]
Team Name: InfoCrew
Team Members: 
Aastha Prasad - mausam20
Tallapaneni Venkateshwara Chowdary - tallapaneniv
Trayvion White-Austin - Twhite13

Project Goal
The project aims to analyze different gradient descent optimizers used in machine learning and deep learning models. this is to visualize how all the optimizers converge differently on a given dataset.

Dataset
They are going to synthetize data to reflect different optimization landscapes, to train the model and get the data for optimizer they are going to use the Paris 2024 Olympics data.

Approach
The approach is to train a neural network model on a dataset multiple times with different optimizer function and collect the optimizer data how it converges, they will create visualizations with animations to view how different optimizers converges.

Clarity
How they will create the optimizer data is not clear.
The questions are specific to ML so for non ML people need to explain the background.

Suggestions for Improvement
write now they are using only one dataset to train the model and get data, can use different train data to visualize which optimizers work better for which data.
They can also including hyperparameters tuning to see how optimizer changes which different hyperparameters like learning rate, batch size. 

Interesting Aspects
The animation part is interesting as we can see how the loss converges visually.

Code Organization
The code should be modular with proper documentation,
As NN is not known to everyone, can describe the forward and backward pass, so that they can understand the loss and optimization process

Additional Feedback
The project looks great overall
